Fakebook.Posts/Fakebook.Posts.Domain/Models/Comment.cs

```diff
@@ -7,16 +7,48 @@ namespace Fakebook.Posts.Domain.Models
     public class Comment
     {
         public int Id { get; set; }
-        public string Content { get; set; }
+        private string _Content; 
         public Post Post { get; set; }
-        public string UserEmail { get; private set; }
+        private string _UserEmail;
         public DateTime CreatedAt { get; set; }
         public ICollection<string> Likes { get; set; }
+        
+
+        public string UserEmail{
+            get{
+                return _UserEmail;
+            }
+            set{
+                if(value.Contains('@')){
+                    _UserEmail = value;
+                }
+                else{
+                    throw new ArgumentException("A correct email format is required.");
```

Comment:
use the parameter name argument with argumentexception as well - nameof(value) in the case of a property setter like this.
check against null first as well with argumentnullexception.
i guess it's fine to move the validation out of the constructor into the property setters, but do the setters need to be public? i guess for the content, sure, we could edit that. but the user email?